SaltLakeCity.md

```diff
@@ -0,0 +1,5 @@
+# SLC, UT
+
+## Where to get baptized
+
+### Where marry ten women at once
```

Comment:
anywhere in utah. they just decriminalized polygamy while the state was in an uproar about gay marriage... sneaky polygamists  :)